app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+
+    storage = {}
+
+    def inner(*args) -> Callable:
```

Comment:
what if I will call function with named parameters?
add **kwargs
provide typehint for *args and **kwargs

Reply:
Using kwargs is not required in this task. But you should still provide annotation for args

Reply:
> That was in readme:
> "Decorator cache creating for decorating only functions that take immutable arguments."
named args != immutable args

Reply:
long_time_func(a=1, b=2, c=3) will raise an error without **kwargs handling
just to know